### DISCUSSION TITLE
10.6.10 state of play...

### DISCUSSION THREAD

**Discussion by TomThistlethwaite** [CommandPost/FCPCafe#261] [Final Cut Pro Questions]
Hey all, 

Just wondered how everyone is getting on with 10.6.10 and if after the recent instability we've got a version that's more or less stable? 

Cheers,

Tom

**Comment by latenitefilms** [comment]
I'm still on 10.6.8 and macOS Monterey on my day-to-day machine for now.

**Reply by TomThistlethwaite** [reply]
Thanks Chris!

**Comment by benblaser60** [comment]
Sonoma 14.1.1 with FCPX 10.6.10 and no real issues. Works like a charm on an M1 MBP and an Intel iMac Pro.

**Comment by simplevideoedit** [comment]
So far I am using Sonoma 14.1 and Final Cut Pro 10.6.10 on my iMac Pro and M1 Macbook Air and things have been pretty smooth. A couple instances of crashes but no more than I was usually having. Looking forward to 10.7!

**Comment by TomThistlethwaite** [comment]
Thanks all - I ended up upgrading to Ventura 13.6.1 and with FCP 10.6.10 in a rare break between delivered projects and it's all working OK. All a relief after that wonky moment!
